.packit.yml

```
@@ -0,0 +1,17 @@
+specfile_path: anaconda.spec
+upstream_package_name: anaconda
+actions:
+  post-upstream-clone:
+    - ./autogen.sh
+    - ./configure
+  get-current-version:
+    - bash -c "git describe --tags --match '*.*' | awk -F '-' '{ print $2}' "
+  create-archive:
+    - "make release"
```

```suggestion
    - "make release"
    - 'bash -c "tar -tf *.tar.bz2 | tail"'
```
let's make sure the tarball is valid, `tail` will make the output brief